tests/unit/test_image_filters.py

```diff
@@ -103,7 +103,7 @@ def it_calls_adaptive_equalization_functional(self, request):
     def it_calls_local_equalization_functional(self, request):
         image = PILImageMock.DIMS_500X500_RGBA_COLOR_155_249_240
         F_local_equalization = function_mock(
-            request, "src.histolab.filters.image_filters_functional.local_equalization",
```

Comment:
I think it's Black which is adding the trailing comma here. What are your settings?

Reply:
the one in our project.toml:
```
[tool.black]
line-length = 88
include = '\.pyi?$'
exclude = '''
/(
    \.git
  | \.hg
  | \.mypy_cache
  | \.tox
  | \.venv
  | _build
  | buck-out
  | build
  | dist
)/
'''
```